endpoints.html

```diff
@@ -9,6 +9,27 @@
   <p>All input strings should be UTF-8 encoded, our response is always UTF-8 encoded.</p>
   <p>Some of our endpoints return both the status of the transaction and signer activities. Further details about the statuses and activities can be found <a href="/status-activity/">here</a>.</p>
 
+  <h3>Default Headers</h3>
+  <p>In order to communicate with Signhost, the API is expecting the following http headers for all requests. The parts between curlybraces indicate values that need to be replaced.</p>
```

Comment:
This would result in saying we require the json version header in every request. But not all requests return JSON, so I don't think this is correct